pkg/url/url.go

```diff
@@ -102,8 +98,8 @@ func getProtocol(route routev1.Route) string {
 }
 
 // GetURLString returns a string representation of given url
-func GetURLString(url URL) string {
-	return url.Protocol + "://" + url.URL
+func GetURLString(protocol, URL string) string {
+	return protocol + "://" + URL
```

Comment:
wouldn't the URL have the protocol @surajnarwade ?

Reply:
I didn't get this ?

Reply:
oh I mis-read the `URL` here as the struct `URL`

Reply:
resolving